build_stage1.sh

```diff
@@ -39,7 +39,11 @@ cd ${BuildDir}
 cp -f ${GitDir}/rhel-atomic-rebuild.repo /etc/yum.repos.d/
 yum -y install ostree rpm-ostree glib2 docker libvirt epel-release libgsystem
 
-cp -f ${GitDir}/atomic7-testing.repo /etc/yum.repos.d/
+
+if [ -f ${GitDir}/atomic7-testing.repo ]; then
```

Comment:
I don't quite understand this your PR sounds like we don't want it.  So what's the goal of this conditional?  Under what scenario *do* we want to use it?